test/ventilation_test.h

```diff
@@ -6,3 +6,5 @@
 int startVentilation_afterOneInspiration_openInspiratoryValve();
 
 int startVentilation_duringInspiration_doesNotopenInspiratoryValve();
+
+int calculateBreathCycleDuration_15BPM();
```

Comment:
make sure to make a complete set of tests as soon as you finish the implementation with the selected RR